swift/mt940.go

```diff
@@ -15,6 +15,7 @@ import (
 
 // MT940 represents a S.W.I.F.T. Transaction Report
 type MT940 struct {
+	ReferenceDate        time.Time
```

Comment:
when I read it correctly you're using `ReferenceDate` as a way for dependency injection. As the value is always hardcoded to `time.Now` at `element.swift.go:30`. What do you think about not exposing it yet and guarding against zero value within the `Unmarshal` method at `swift/mt940_unmarshaler.go:12`?